lib/_fields.js

```diff
@@ -232,6 +232,10 @@ var fieldDefs = {
     type: String,
     allowedValues: ["Weighted", "Unweighted", "Don't know"]
   },
+  feelingAboutApplication: {
```

Comment:
Every other field def uses "names_with_underscores", not "camelCaseNames", because of legacy code that started with underscores.  While it'd probably be best for us to switch everything to camel case for consistency with javascript, it's worse to mix conventions.  I'd prefer to rename this `"feeling_about_application"` until we take it upon ourselves to rename everything.